widgetry/src/geom/mod.rs

```diff
@@ -267,22 +270,26 @@ impl GeomBatch {
         self
     }
 
-    /// Exports the batch to a list of GeoJSON features, labeling each colored polygon. Z-values,
-    /// alpha values from the color, and non-RGB fill patterns are lost. If the polygon isn't a
-    /// ring, it's skipped. The world-space coordinates are optionally translated back to GPS.
+    /// Exports the batch to a list of GeoJSON features, labeling each colored triangle. Note the
```

Comment:
This is only used by a debug mode in A/B Street. It's kind of a useless feature anyway. Now that rendering detailed lane markings are being moved to [osm2streets](https://github.com/a-b-street/osm2streets/blob/205948f74e14c335cf6213f4478ae9d8d7e34817/street_network/src/render.rs#L94), the original use case for this kind of thing is going away anyway